apps/room/tests/test_views/test_room_share_button.py

```
@@ -0,0 +1,60 @@
+        response = self.client.get(reverse("room:detail", kwargs={"room_slug": self.room.slug}))
+
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, "Claim your invitation")
+        self.assertContains(response, "Create a free account")
+
+    def test_who_are_you_without_guests_promotes_account_creation(self):
+        self.client.logout()
+
+        response = self.client.get(reverse("room:detail", kwargs={"room_slug": self.room.slug}))
+
+        self.assertEqual(response.status_code, 200)
```

Ruff (0.14.6)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)